types/resize-observer-browser/test/resize-observer-browser-module.tests.ts

```diff
@@ -1,5 +1,3 @@
-import { ResizeObserver } from "resize-observer-browser";
-
 function resizeObserverCreates(): void {
     const resizeObserver: ResizeObserver = new ResizeObserver((entries) => {
```

Comment:
We should add a test in the scenario where the callback is complex and so is its own function.

Something like

```
function callback(entries: TheType): void {
  // no-op
}

const resizeObserver = new ResizeObserver(callback);
```

This way we know `TheType` is usable without needing to be imported.

Reply:
Done.